microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/core/PropertiesManager.java

```diff
@@ -509,10 +510,13 @@ public IMember findDeclaredProperty(IJavaProject javaProject, String sourceType,
 				// Method signature has been generated with JDT API, so we are sure that we have
 				// a ')' character.
 				int endBracketIndex = sourceMethod.indexOf(')');
-				String methodSignature = sourceMethod.substring(startBracketIndex + 1, endBracketIndex);
+				String methodSignature = sourceMethod.substring(startBracketIndex, endBracketIndex + 1);

```

Comment:
why did you change this code?

Reply:
For calling `Signature.getParameterTypes(methodSignature)`, the method signature needed the `(` and `)` around it. I changed this code because previously, the brackets were not being included in the substring

Reply:
It was for go to definiton to work